docs/sql-ref-syntax-aux-show-tables.md

```diff
@@ -18,5 +18,73 @@ license: |
   See the License for the specific language governing permissions and
   limitations under the License.
 ---
+### Description
 
-**This page is under construction**
+List all the `tables` from the database with `database-name` and `is temporary table`.
+
+### Syntax
+{% highlight sql %}
+SHOW TABLES [{FROM|IN} database_name] [LIKE 'regex_pattern']
+{% endhighlight %}
+
+### Parameters
+<dl>
+  <dt><code><em>{FROM|IN} database_name</em></code></dt>
+  <dd>
+     Specifies the `database` name from which tables are listed.
+  </dd>
+  <dt><code><em>LIKE 'regex_pattern'</em></code></dt>
+  <dd>
+     Specifies the regex pattern that is used to filter out unwanted tables.
+       <br> - Only `*` and `|` are allowed as wildcard pattern.
```

Comment:
@shivusondur Can you please include the example with output using the 'I' as wildcard ? From the code its not clear to me if this is supported.

Reply:
done